include/deal.II/grid/grid_tools.h

```diff
@@ -797,13 +797,17 @@ namespace GridTools
     const std::vector<std::vector<BoundingBox<spacedim>>> &global_bboxes);
 
   /**
-   * Return a map of index:Point<spacedim>, containing the used vertices of the
-   * given `container`. The key of the returned map is the global index in the
-   * triangulation. The used vertices are obtained by looping over all cells,
+   * Return a map `vertex index -> Point<spacedim>`, containing the used
+   * vertices of the given `container`. The key of the returned map (i.e.,
```

Comment:
The use of `container` is unusual here since we don't allow DoFHandlers. Would you mind switching the argument to `tria` and this to `@p tria` too?

Reply:
hm, why do you think we don't allow `DoFHandler` objects?